### DISCUSSION TITLE
Support for GCC

### DISCUSSION THREAD

**Discussion by snydertl** [microsoft/PowerPlatformConnectors#1559] [Q&A]
Hi all, 

I am trying to login to our GCC environment but am not finding a way to do so.  When I do the paconn login, it takes me to a commercial side (we had some issues when we first had our environment setup). Microsoft support suggested connecting using the pac auth command but that is failing to connect to our tenant as well.  

Any suggestions would be appreciated!

Tina Snyder

**Comment by Amjed-Ayoub** [comment]
Hi @snydertl, thank you for reaching out.
May I ask you please to open an email thread with us and I can loop in the appropriate team to help. 
You can use our email "ConnectorPartnerMgmtTeam@service.microsoft.com" :-).
Thank you very much.
